poly-commitment/src/commitment.rs

```diff
@@ -879,6 +886,35 @@ pub fn inner_prod<F: Field>(xs: &[F], ys: &[F]) -> F {
     res
 }
 
+/// Cast the evaluations in specfic domain size to the smaller domain size.
+///
+/// ## Panics
+///
+/// Panics if `evals_domain_size` is smaller than `target_domain_size`.
+pub fn to_domain<F: FftField>(
+    evals: &Evaluations<F, D<F>>,
+    evals_domain_size: usize,
+    target_domain_size: usize,
```

Comment:
can't you just do `target_domain.size()`?

Reply:
same for `evals_domain_size`

Reply:
yeah but since you use them to compute the ratio, they should give you a similar result

Reply:
btw this part of the mina book is relevant: https://o1-labs.github.io/proof-systems/plonk/domain.html

(and so is https://github.com/duguorong009/proof-systems/blob/create-move-domain-size-helper/kimchi/src/circuits/domains.rs)